### PR TITLE
Merge new console

### PR DESCRIPTION
**Pull request by gemmai95**
Add new console2 features to master.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # sdx-console
 
-[![Build Status](https://travis-ci.org/ONSdigital/sdx-console.svg?branch=master)](https://travis-ci.org/ONSdigital/sdx-console) [![codecov](https://codecov.io/gh/ONSdigital/sdx-console/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/sdx-console)
```
Comment (dcdarrell9) [inline README.md:3]: Is the codecov badge still in the Readme?

File: .coverage [hunk 2]
```diff
@@ -0,0 +1 @@
...2,616,617,620,624,625,627,633,634,637,638,641,642,645,646,649,650,653,654,657,658,661,662,665,666,669,686,688,690,697,700,702,704,713,723,726,738,740,743,755,757,760,770,771,774,784,785,788,798,799,802,812,813,293,299,301,302,303,306,307,309,311,313,314,318,319,320,321,532,533,534,535,536,537,539,540,541],"/Users/millaa/.virtualenvs/sdx-consoleenv/lib/python3.6/site-packages/sqlalchemy/sql/sqltypes.py":[10,12,13,14,15,17,18,19,21,22,23,24,25,26,27,28,30,34,42,44,48,49,51,59,62,65,67,69,80,83,91,93,95,98,107,110,122,124,129,203,209,241,261,268,272,282,283,286,340,342,344,358,369,371,373,387,389,391,393,396,400,405,435,442,444,447,454,456,459,497,499,501,504,554,563,566,571,578,584,609,636,655,657,659,662,698,706,728,745,747,749,764,767,771,784,786,788,790,793,797,822,824,826,828,831,834,838,852,854,856,859,865,871,887,901,908,916,920,928,930,932,944,946,948,954,975,978,1000,1003,1006,1014,1055,1058,1070,1074,1083,1092,1100,1108,1116,1124,1136,1204,1206,1208,1312,1324,1337,1356,1358,1367,1369,1376,1382,1388,1408,1411,1434,1444,1454,1467,1475,1486,1488,1490,1491,1511,1516,1532,1548,1555,1562,1564,1567,1582,1587,1604,1608,1617,1623,1630,1646,1648,1649,1653,1675,1685,1689,1704,1720,1743
```
Comment (dcdarrell9) [inline .coverage]: What is this file for? And I can see Andrews user path in it

File: console/helpers/exceptions.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class ResponseError(Exception):
+    def __init__(self, url, message, status_code):
+        self.url = url
+        self.message = message
+        self.status_code = status_code
+
+
+class ClientError(ResponseError):
+    pass
```
Comment (dcdarrell9) [inline console/helpers/exceptions.py:9]: Should these `pass` or should they be doing something?

File: console/settings.py [hunk 4]
```diff
@@ -1,63 +1,37 @@
-import os
 import logging
+import os
```
Comment (dcdarrell9) [inline console/settings.py:3]: isn't a lot of this already in config.py? Or the other way around 🙃

File: console/static/js/get_links.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+function genList(page, noPages) {
```
Comment (dcdarrell9) [inline console/static/js/get_links.js:1]: Is this no pages or number of pages?

**Review by warrenbailey — CHANGES_REQUESTED** [review]
I don't want this merged into master yet

File: console/views.py [hunk 6]
```diff
@@ -1,322 +1,251 @@
...
-
-@app.route('/validate', methods=['POST', 'GET'])
-def validate():
+        return render_template('decrypt.html', current_user=flask_security.core.current_user)
+
+
+def get_filtered_responses(logger, valid, tx_id, ru_ref, survey_id, datetime_earliest, datetime_latest):
+    logger.info('Retrieving responses from sdx-store')
+    try:
+        q = db_session.query(SurveyResponse)
```
Comment (dcdarrell9) [inline console/views.py]: I'd say to have something else here instead of 'q', but not a problem

**Pull request closed without merging**